workloads/kube-burner/common.sh

```diff
@@ -142,6 +136,9 @@ check_running_benchmarks() {
 }
 
 cleanup() {
+  log "Cleaning up benchmark"
+  kubectl delete -f ${TMPCR}
+  kubectl delete configmap -n benchmark-operator kube-burner-cfg-${UUID}
   if ! oc delete ns -l kube-burner-uuid=${UUID} --grace-period=600 --timeout=30m; then
```

Comment:
if we're doing this , should we close this pr?

Reply:
yep